pymc/model.py

```diff
@@ -510,7 +510,6 @@ def __init__(self, mean=0, sigma=1, name=''):
             CustomModel(mean=2, name='second')
 
         # variables inside both scopes will be named like `first::*`, `second::*`
-
```

Comment:
Removed line by accident?

Reply:
No, the rest of the code does introduce an extra blank line at the end of the docstring.